### PR TITLE
test(elevation): unit test for reading preprocessed data

### PR DESCRIPTION
**Pull request by takb**
Add unit test for reading preprocessed elevation data from OSM file

### Pull Request Checklist

- [x] 1. I have [**rebased**][rebase] the latest version of the master branch into my feature branch and all conflicts
         have been resolved.
- [ ] 2. I have added information about the change/addition to functionality to the CHANGELOG.md file under the
         [Unreleased] heading.
- [ ] 3. I have documented my code using JDocs tags.
- [ ] 4. I have removed unnecessary commented out code, imports and System.out.println statements.
- [ ] 5. I have written JUnit tests for any new methods/classes and ensured that they pass.
- [ ] 6. I have created API tests for any new functionality exposed to the API.
- [ ] 7. If changes/additions are made to the ors-config.json file, I have added these to the [ors config documentation][config]
         along with a short description of what it is for, and documented this in the Pull Request (below).
- [x] 8. I have built graphs with my code of the Heidelberg.osm.gz file and run the api-tests with all test passing
- [ ] 9. I have referenced the Issue Number in the Pull Request (if the changes were from an issue).
- [ ] 10. For new features or changes involving building of graphs, I have tested on a larger dataset
          (at least Germany), and the graphs build without problems (i.e. no out-of-memory errors).
- [ ] 11. For new features or changes involving the graphbuilding process (i.e. changing encoders, updating the
          importer etc.), I have generated longer distance routes for the affected profiles with different options
          (avoid features, max weight etc.) and compared these with the routes of the same parameters and start/end
          points generated from the current live ORS.
          If there are differences then the reasoning for these **MUST** be documented in the pull request.
- [ ] 12. I have written in the Pull Request information about the changes made including their intended usage
          and why the change was needed.
- [ ] 13. For changes touching the API documentation, I have tested that the API playground [renders correctly][api].

### CODE REVIEW

File: openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java [hunk 1]
```diff
@@ -45,6 +50,50 @@ void directRouteTest() {
 
     }
 
+    @Test
+    public void buildGraphWithPreprocessedData() {
```
Comment (MichaelsJP) [inline openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java]:
```suggestion
    void buildGraphWithPreprocessedData() {
```

File: openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java [hunk 2]
```diff
@@ -45,6 +50,50 @@ void directRouteTest() {
...
+        //  </node>
+        // => NaN, elevation will be set to 0
+        //  <node id="...">
+        //    <tag k="ele" v="1,912.1"/>
+        //  </node>
+        // => 1912.1
+        //  <node id="...">
+        //    <tag k="ele" v="1.021,12"/>
+        //  </node>
+        // => 1.02112
```
Comment (MichaelsJP) [inline openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java]:
```suggestion
```

File: openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java [hunk 3]
```diff
@@ -45,6 +50,50 @@ void directRouteTest() {
 
     }
```
Comment (MichaelsJP) [inline openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java:52]: /**
     * This tests loading an OSM dataset preprocessed with ors-preprocessor, as used on our production servers.
     * For testing purposes, a subset from the Heidelberg graph has been modified with several invalid ele tag
     * values to demonstrate that the graph will still build, albeit with incorrect but valid elevation values.
     * <node id="...">
     *   <tag k="ele" v="invalid ele tag"/>
     * </node>
     *=> NaN, elevation will be set to 0
     * <node id="...">
     *   <tag k="ele" v="198.0.0.4"/>
     * </node>
     *=> NaN, elevation will be set to 0
     * <node id="...">
     *   <tag k="ele" v="1,912.1"/>
     * </node>
     *=> 1912.1
     * <node id="...">
     *   <tag k="ele" v="1.021,12"/>
     * </node>
     *=> 1.02112
     */

**Review by MichaelsJP — CHANGES_REQUESTED** [review]

**Review by MichaelsJP — CHANGES_REQUESTED** [review]
Please rename the `orspp.pbf` to e.g.  `ors_elevation_pre_processed.pbf`

File: openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java (hunk 3 above)
Comment (takb) [reply openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperTest.java:52]: moved

**Comment by takb** [comment]
> Please rename the `orspp.pbf` to e.g. `ors_elevation_pre_processed.pbf`

done

**Review by MichaelsJP — APPROVED** [review]

**Pull request merged**